config-linux.md

```diff
@@ -490,9 +490,9 @@ For more information, see [the man page](http://man7.org/linux/man-pages/man8/sy
    }
 ```
 
-## seccomp
+## Seccomp
 
-Seccomp provides application sandboxing mechanism in the Linux kernel.
+**`seccomp`** (object, OPTIONAL) provides application sandboxing mechanism in the Linux kernel.
```

Comment:
> One of the other "Seccomp" references should be changed.

+1 to not changing this instance, but I don't think there is another reference which should be changed instead.  I think the seccomp section is just missing a Markdown spec, so we need to add completely new language along the lines of:

```
* **`seccomp`** (object, OPTIONAL)

    The following parameters can be specified to setup seccomp:

    * **`defaultAction`** (string, REQUIRED) …something about what this means…

        Runtimes MUST support at least the following values:

        * `SCMP_ACT_KILL`
        * `SCMP_ACT_TRAP`
        …
    …
```